CHANGELOG/CHANGELOG-1.3.md

```diff
@@ -3,7 +3,108 @@
 <hr>
 
 
-## [v1.3.6](https://github.com/aws/aws-k8s-tester/releases/tag/v1.3.6) (2020-06)
```

Comment:
can you rebase and remove all changes from current upstream master branch?

Reply:
Sorry, what do you mean by remove all changes? Should I revert back to before the merged upstream master into my branch? Or pull down the new changes currently on upstream master?

Reply:
The commit should exclude merge commit. Should be only diff between your branch and current master, to make it easier to review.